<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LiczKalorie</class>
 <widget class="QDialog" name="LiczKalorie">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Licz kalorie</string>
  </property>
  <widget class="QListWidget" name="listWidget_produkty">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>281</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_kalorie">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>151</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_suma">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>441</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>19</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Produkt&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>326</x>
     <y>19</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Kalorie&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>270</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Suma kalorii&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
